process/profile.go

```diff
@@ -38,6 +39,16 @@ func (p *Process) GetProfile(ctx context.Context) (changed bool, err error) {
 		profileID = profile.SystemProfileID
 	case ownPID:
 		profileID = profile.PortmasterProfileID
+	default:
+		// Check if this is another Portmaster component.
+		if updatesPath != "" && strings.HasPrefix(p.Path, updatesPath) {
+			switch {
```

Comment:
Maybe consider using `filepath.ToSlash()` to ensure the comparision uses the same separator style (`/` is also a valid separator on windows).
Also, the `updatesPath != ""` should be `updatesPath != "/"` as that's the "empty" value. See the issue in process/module.go

I.e. 
```suggestion
		// Check if this is another Portmaster component.
		if strings.HasPrefix(filepath.ToSlash(p.Path), filepath.ToSlash(updatesPath)) {
			switch {
```